server/service/async/async_test.go

```diff
@@ -7,409 +7,188 @@ import (
 	"time"
 
 	"github.com/fleetdm/fleet/v4/server/datastore/mysql"
-	"github.com/fleetdm/fleet/v4/server/datastore/redis"
```

Comment:
This file got reorganized and now contains the entrypoints to the async tests, dispatching to label/policy tests in their respective files.